e2e/app/component/help-sidebar.ts

```diff
@@ -10,7 +10,7 @@ import {ElementType} from 'app/xpath-options';
 import {waitWhileLoading} from 'utils/test-utils';
 import {waitForNumericalString} from 'utils/waits-utils';
 
-const defaultXpath = '//*[@id="help-sidebar"]';
+const defaultXpath = '//*[@id="help-sidebar"]//*[@id="selection-list"]';
```

Comment:
The help-sidebar is not exclusively used for cohorts and `#selection-list` is specific to cohorts, so this is not a reasonable default. It may cover all current use cases, but if so it's because cohorts are the only test coverage we have involving the help sidebar.

I would add another helper like `selectionXpath` or `cohortSelectionXpath`and use that from individual methods instead. In general, it would also be nice if the identifiers were named more specifically, but this is an issue with the element IDs as well.